server/builtin.go

```diff
@@ -1,29 +1,41 @@
 package server
 
 import (
+	"github.com/0xPolygon/polygon-edge/chain"
 	"github.com/0xPolygon/polygon-edge/consensus"
 	consensusDev "github.com/0xPolygon/polygon-edge/consensus/dev"
 	consensusDummy "github.com/0xPolygon/polygon-edge/consensus/dummy"
 	consensusIBFT "github.com/0xPolygon/polygon-edge/consensus/ibft"
+	consensusPolyBFT "github.com/0xPolygon/polygon-edge/consensus/polybft"
 	"github.com/0xPolygon/polygon-edge/secrets"
 	"github.com/0xPolygon/polygon-edge/secrets/awsssm"
 	"github.com/0xPolygon/polygon-edge/secrets/gcpssm"
 	"github.com/0xPolygon/polygon-edge/secrets/hashicorpvault"
 	"github.com/0xPolygon/polygon-edge/secrets/local"
+	"github.com/0xPolygon/polygon-edge/types"
 )
 
+type InitGenesisFn func(*chain.Chain, map[types.Address]*chain.GenesisAccount) (
+	map[types.Address]*chain.GenesisAccount, error)
+
 type ConsensusType string
 
 const (
-	DevConsensus   ConsensusType = "dev"
-	IBFTConsensus  ConsensusType = "ibft"
-	DummyConsensus ConsensusType = "dummy"
+	DevConsensus     ConsensusType = "dev"
+	IBFTConsensus    ConsensusType = "ibft"
+	PolyBFTConsensus ConsensusType = "polybft"
+	DummyConsensus   ConsensusType = "dummy"
 )
 
 var consensusBackends = map[ConsensusType]consensus.Factory{
-	DevConsensus:   consensusDev.Factory,
-	IBFTConsensus:  consensusIBFT.Factory,
-	DummyConsensus: consensusDummy.Factory,
+	DevConsensus:     consensusDev.Factory,
+	IBFTConsensus:    consensusIBFT.Factory,
+	PolyBFTConsensus: consensusPolyBFT.Factory,
+	DummyConsensus:   consensusDummy.Factory,
+}
+
+var consensusInitGenesis = map[ConsensusType]InitGenesisFn{
```

Comment:
Let's not do this, otherwise, we will have a factory for everything we want to configure on the client. If possible, I would create a function in `Consensus` interface that modifies the `Genesis` file instead.

Reply:
yes we will have separated command at first and then we will abstract existing genesis command